boundary/analyze.py

```diff
@@ -212,7 +212,13 @@ def check_redirect_mangled(f, meta):
     config['sdcr_srcs'] = [f[1] for f in config['sidecar']]
     config['all_files'] = config['mod_hdrs'] + config['mod_srcs'] + config['sdcr_srcs']
     config['fullname']  = {os.path.basename(f):f for f in config['all_files']}
-    metas = list(map(read_meta, all_meta_files()))
+
+    metas = []
+    metas_by_name = {}
+    for f in all_meta_files():
```

Comment:
For these three variable names f, fn and file, we use it freely but sometimes confused.